spec/components/dashboard/continue_deposit_modal_component_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Dashboard::ContinueDepositModalComponent, type: :component do
+  let(:rendered) { render_inline(described_class.new(presenter: presenter)) }
+  let(:work_version) { build_stubbed(:work_version) }
+  let(:collection_version) { build_stubbed(:collection_version) }
```

Comment:
Can you move these into the contexts where they are used?

Reply:
Yes, I actually added another test as well to ensure we capture the three scenarios (only a work in progress, only a collection in progress, both collection and work in progress).